.travis.yml

```diff
@@ -8,13 +8,10 @@ python:
 dist: xenial
 
 before_install:
-    - wget 'https://github.com/cylc/cylc-flow/archive/7.8.x.tar.gz' -O '/tmp/cylc-master.tar.gz'
-    - tar -xvf '/tmp/cylc-master.tar.gz' -C "${HOME}"
-    - mv "${HOME}/cylc-flow-7.8.x" "${HOME}/cylc-master"
-    - make -C "${HOME}/cylc-master" 'version'
-    - wget 'https://github.com/metomi/fcm/archive/master.tar.gz' -O '/tmp/fcm-master.tar.gz'
-    - tar -xvf '/tmp/fcm-master.tar.gz' -C "${HOME}"
-    - cat >"${HOME}/.bashrc" <<<"export PATH=${PWD}/bin:${HOME}/fcm-master/bin:${HOME}/cylc-master/bin:\$PATH"
+    - wget 'https://github.com/cylc/cylc-flow/archive/7.8.x.tar.gz' -O - | tar -C "${HOME}" -xz
+    - make -C "${HOME}/cylc-flow-7.8.x" 'version'
+    - wget 'https://github.com/metomi/fcm/archive/master.tar.gz' -O - | tar -C "${HOME}" -xz
+    - echo "export PATH=${PWD}/bin:${HOME}/fcm-master/bin:${HOME}/cylc-flow-7.8.x/bin:\$PATH" >>"${HOME}/.bashrc"
```

Comment:
Note behaviour change here as we are now appending to the bashrc rather than re-writing it (which makes sense).